SignalMessaging/categories/Theme.m

```diff
@@ -49,7 +49,7 @@ + (void)setIsDarkThemeEnabled:(BOOL)value
 
 + (UIColor *)backgroundColor
 {
-    return (Theme.isDarkThemeEnabled ? UIColor.ows_blackColor : UIColor.ows_whiteColor);
+    return (Theme.isDarkThemeEnabled ? UIColor.ows_gray95Color : UIColor.ows_whiteColor);
```

Comment:
Is this right?

I see "Dark: **Mobile** Background" should be "#000000"
Whereas gray95 is for "Dark: **Desktop** Background"

Reply:
This was my only real concern - feel free to merge after addressing.

Reply:
This is pending a final decision from Myles - he's still mulling - but this is supposedly the new design.